### PR TITLE
Postgres code review fix

### PR DESCRIPTION
**Pull request by sureshdash2022-yb**
@suranjan @vaibhav-yb  please review the code changes

### CODE REVIEW

File: src/postgres/src/backend/utils/adt/arrayfuncs.c [hunk 1]
```diff
@@ -1045,50 +1045,48 @@ array_out(PG_FUNCTION_ARGS)
...
-	  my_extra->element_type = ~element_type;
-	}
-
-	if (my_extra->element_type != element_type)
+		decode_options = (DatumDecodeOptions *) PG_GETARG_POINTER(1);
+		typlen = decode_options->elem_len;
+		typbyval = decode_options->elem_by_val;
+		typalign = decode_options->elem_align;
+		typdelim = decode_options->elem_delim;
+	} else
```
Comment (suranjan) [inline src/postgres/src/backend/utils/adt/arrayfuncs.c:1055]: is this else correct?

File: src/postgres/src/include/ybgate/ybgate_api.h [hunk 2]
```diff
@@ -86,11 +86,11 @@ typedef MemoryContext YbgMemoryContext;
 YbgStatus YbgGetCurrentMemoryContext(YbgMemoryContext *memctx);
 
 YbgStatus YbgSetCurrentMemoryContext(YbgMemoryContext memctx,
-									 YbgMemoryContext *oldctx);
+		YbgMemoryContext *oldctx);
```
Comment (suranjan) [inline src/postgres/src/include/ybgate/ybgate_api.h:89]: was this our change?

File: src/postgres/src/include/ybgate/ybgate_api.h [hunk 3]
```diff
@@ -86,11 +86,11 @@ typedef MemoryContext YbgMemoryContext;
...
 
 YbgStatus YbgCreateMemoryContext(YbgMemoryContext parent,
-								 const char *name,
-								 YbgMemoryContext *memctx);
+		const char *name,
```
Comment (suranjan) [inline src/postgres/src/include/ybgate/ybgate_api.h:92]: was this our change?

**Review by suranjan — CHANGES_REQUESTED** [review]
please check again

**Review by sureshdash2022-yb — COMMENTED** [review]
@suranjan please review it.

File: src/postgres/src/backend/utils/adt/arrayfuncs.c (hunk 1 above)
Comment (sureshdash2022-yb) [reply src/postgres/src/backend/utils/adt/arrayfuncs.c:1055]: Fixed

**Review by sureshdash2022-yb — COMMENTED** [review]
@suranjan please review those fix

**Comment by vaibhav-yb** [comment]
This PR is raise against the master, it should be against our branches, right?

**Comment by sureshdash2022-yb** [comment]
> This PR is raise against the master, it should be against our branches, right?

This was just for temporary check. to know thing we have added against the master w.r.t postgres side.

**Review by suranjan — APPROVED** [review]

**Pull request merged**